webpay/pay/views.py

```diff
@@ -181,321 +181,6 @@ def configure_transaction(request, data=None):
             'payment_required': payment_required}
 
 
-def index(request):
-    """Hand off either lobby or serving Spartacus depending on settings."""
-    if settings.SPA_ENABLE:
-        from webpay.spa.views import index as spa_index
-        return spa_index(request)
-    else:
-        return lobby(request)
-
-
-@anonymous_csrf_exempt
-@require_GET
-def lobby(request):
-    sess = request.session
-    have_jwt = bool(request.GET.get('req'))
-
-    log.info('starting from JWT? {have_jwt}'.format(have_jwt=have_jwt))
-    if have_jwt:
-        # If it returns a response there was likely
-        # an error and we should return it.
-        res = process_pay_req(request)
-        if isinstance(res, http.HttpResponse):
-            return res
-    elif settings.TEST_PIN_UI:
-        # This won't get you very far but it lets you create/enter PINs
-        # and stops a traceback after that.
-        sess['trans_id'] = trans_id()
-
-    pin_form = VerifyPinForm()
-
-    if sess.get('uuid'):
-        auth_utils.update_session(request,
-                                  sess.get('uuid'),
-                                  False,
-                                  request.session.get('logged_in_user', None))
-
-        redirect_url = check_pin_status(request)
-        if redirect_url is not None:
-            return http.HttpResponseRedirect(
-                '{0}?next={1}'.format(reverse('pay.bounce'), redirect_url)
-            )
-
-    # If the buyer closed the trusted UI during reset flow, we want to unset
-    # the reset pin flag. They can hit the forgot pin button if they still
-    # don't remember their pin.
-    if sess.get('uuid_needs_pin_reset'):
-        try:
-            solitude.set_needs_pin_reset(sess['uuid'], False)
```

Comment:
I don't think `solitude.set_needs_pin_reset` is used anywhere now, lets remove that?